dist/bin/current/helperScripts/downloadFunctions.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+function checkAndDownloadGroovyServ() {
+  # Check for groovyserv. If it does not exist, try to download it. If it still fails, start Roddy without it.
+  local callerBinary=java
+  local roddyApplicationDirectory=${1-""}
+  local runtimeFolder=${roddyApplicationDirectory}/dist/runtime
+  local forbiddenFile=${runtimeFolder}/gservforbidden
+
+  if [[ ! -d $(find ${runtimeFolder} -maxdepth 1 -mindepth 1 -type d -name "groovys*") && ! -f ${forbiddenFile} ]]; then
+
+    mkdir -p dist/runtime/ 2>/dev/null
+    (cd ${runtimeFolder} && wget -S $(curl -s https://kobo.github.io/groovyserv/download.html | grep groovyserv | grep bin.zip | sed 's/href=/\n/g' | sed 's/>/\n/g' | sed 's/</\n/g' | sed 's/"//g'  | grep .zip)) &> /dev/null
+
+    if [[ $? -ne 0 ]]; then
+      touch ${forbiddenFile}
```

Comment:
What about issuing a warning to the user via stderr? I know, most users will forget after some time, but some might not. It'd give the feeling that Roddy is not a blackbox, without overloading the user.

Alternatively/Additionally, you may add a warning that the forbiddenfile is used and no groovyserv is used.